src/main/resources/i18n/messages.properties

```diff
@@ -866,49 +863,71 @@ project.samples.filter.by-endDate=To
+NcbiSubmissionRequest.namespace=Identifier Namespace
+NcbiSubmissionRequest.namespace.description=Prefix to use for submission identifiers in NCBI.
+NcbiSubmissionRequest.releaseDate=Release Date
+NcbiSubmissionRequest.releaseDate.description=Submission will not be made public until after the chosen release date.
+
+# TYPE: NcbiBioSample
+NcbiBioSample.id=BioSample ID
+NcbiBioSample.accession=NCBI Accession
+NcbiBioSample.bioSample=BioSample
+NcbiBioSample.libraryName=Library Name
+NcbiBioSample.libraryStrategy=Library Strategy
+NcbiBioSample.librarySource=Library Source
+NcbiBioSample.libraryConstructionProtocol=Library Construction Protocol
+NcbiBioSample.instrumentModel=Instrument Model
+NcbiBioSample.librarySelection=Library Selection
+NcbiBioSample.status=Status
+NcbiBioSample.singles=Single Files
+NcbiBioSample.pairs=Paired Files
+
+# TYPE: NcbiSubmission
+NcbiSubmission.id=Submission ID
+NcbiSubmission.project=Project
+NcbiSubmission.state=Status
+NcbiSubmission.submitter=Submitter
+NcbiSubmission.createdDate=Created Date
+NcbiSubmission.organization=Organization
+NcbiSubmission.bioProject=BioProject
+NcbiSubmission.ncbiNamespace=NCBI Namespace
+NcbiSubmission.releaseDate=Release Date
+NcbiSubmission.bioSamples=Samples
+
+# NEW
+CreateNcbiExport.title=NCBI SRA Export Tool
+CreateNcbiExport.details=Submission Details
+CreateNcbiExport.samples=Samples
+CreateNcbiExport.pairs=Paired End Sequencing Files
+CreateNcbiExport.singles=Single End Sequencing Files
+CreateNcbiExport.files.required=Must select a sequencing file.
+CreateNcbiExport.required=This is a required field.
+CreateNcbiExport.submit=Submit
+CreateNcbiExport.default.title=Default Sample Settings
+CreateNcbiExport.default.description=Values selected here will be used as default values for all samples. Individual samples can overwrite these default values.
+
+
+CreateNcbiExport.biosample-id=BioSample ID
+CreateNcbiExport.biosample-id.desc=An identifier block that lists not only the BioSample accession assigned to that record, but also any other external sample identifier, such as that issued by the source database or repository.
+CreateNcbiExport.library_name=Library Name
+CreateNcbiExport.library_strategy=Library Strategy
+CreateNcbiExport.library_strategy.description=Sequencing technique intended for this library. <a href="http://www.ncbi.nlm.nih.gov/books/NBK54984/table/SRA_Glossary_BK.T._library_descriptor_te/" target="_blank">NCBI Documentation</a>.
+CreateNcbiExport.library_source=Library Source
+CreateNcbiExport.library_construction_protocol=Library Construction Protocol
+CreateNcbiExport.instrument_model=Instrument Model
+CreateNcbiExport.library_selection=Library Selection
+CreateNcbiExport.success=Samples were successfully uploaded to the NCBI Sequence Read Archive.
+CreateNcbiExport.error=There was an unkown error uploading toe the NCBI SRA.  If this persists please contact a system administrator.
```

Comment:
Typo `unkown` and `toe`. Also, it looks like there are is an extra space between the two sentences. Update the second sentence to `If this error persists please contact a system administrator.`

Reply:
Yikes. Guess I was thinking about my toe!  Updated in 8dde4fdda9ad55da3f720be9104f4b8edd651aae

Reply:
Haha! Thanks!